.cascade-code/left-nav-refactor/_components/left_nav/left_nav_index_format

```diff
@@ -0,0 +1,164 @@
+                    </a>
+                    
+                    #if ( $siblingHasChildren )
+                        #displayChildren( $children )
+                    #end
+                    
+                </li>
+                    
+            #else
+            
+                #if ( $_XPathTool.selectSingleNode( $sibling, "system-page[name='index']" ) )
+                    
+                    #set ( $catch = $_XPathTool.selectSingleNode( $sibling, "system-page[name='index']" ) )
+                        
+                    #if ( !$_XPathTool.selectSingleNode( $catch, "dynamic-metadata[1][value='Yes']" ) )
+                        
+                        #if ( $current == true )
+                            <li class="active">
+                        #else
+                            <li>
+                        #end
+
+                        #if ( $siblingHasChildren )
+                            #displayChildren( $children )
+                        #end
+                        </li>
+                            
+                    #end
+                        
+                #end
+                    
+            #end
+
+        #elseif ( $siblingName == "system-page" && $hideSibling == [false] )
+            #if ( $current == true )
+                <li class="active">
+            #else
+                <li>
+            #end
+
+                <a href="${siblingPath}" title="${siblingTitle}" class="s-p">
+                    $siblingTitle 
+                </a>
+            
+            </li>
+        #end
+
+    #end
+    </ul>
+#end
+
+#if ( $currentPageName.value == "index" && $parentName.value == "index" )
+    #set ( $parentPath = $_XPathTool.selectSingleNode($parent, "../..") )
+    #set ( $parentTitle = $_EscapeTool.xml( $parentPath.getChild("display-name").value ) )
+    
+    ### if fails to find parent page, don't do this LI...gives error (eg parent folder unindexed or not right metadata:
+    #if ( $parentPath.getChild("display-name").value )
+        <div class="leftTitle">
+            <ul>
+                <li class="title">
+                <a href="[system-asset]${parentPath.getChild("link").value}/index[/system-asset]" target="_top" title="${parentTitle}">
+                    $parentTitle
+                </a>
+                </li>
+            </ul>
+        </div>
+    #end
+#end
+
+#if ( $current )
+    #set ( $siblings = $_XPathTool.selectNodes($currentPage, "//system-folder[system-page[@current and name = 'index']] | //system-page[@current and name != 'index'] | //system-folder[system-page[@current and name = 'index']]/following-sibling::*[name != 'index'] | //system-folder[system-page[@current and name = 'index']]/preceding-sibling::*[name != 'index'] | //system-page[@current and name != 'index']/following-sibling::*[name != 'index'] | //system-page[@current and name != 'index']/preceding-sibling::*[name != 'index']") )
+    #showPage( $siblings )
+#end
```

Comment:
This is like the `main` routine that builds the menu, yes? Is the previous block just the header?

Reply:
Yes I'd say this was the main block. It's really just the two lines inside though now because I removed the conditional